### PR TITLE
Add `using Random` to `runtests.jl`

### PR DESCRIPTION
**Pull request by staticfloat**
Fixes errors on 0.7+

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -2,6 +2,7 @@ using PkgDev
 using Base.Test
 using Compat
 import Base.Pkg.PkgError
+using Random
```
Comment (ararslan) [inline test/runtests.jl]: Probably has to be `Compat.Random` for 0.6 compatibility

**Comment by staticfloat** [comment]
Now it looks like it's sad it can't find `find_in_path()`

**Comment by ararslan** [comment]
Yeah IIRC that was removed in one of Stefan's Pkg3-related PRs. I'm not sure what the replacement is.

File: test/runtests.jl [hunk 2]
```diff
@@ -3,6 +3,10 @@ using Base.Test
 using Compat
 import Base.Pkg.PkgError
 
+if VERSION >= v"0.6"
+    using Compat.Random
```
Comment (tkelman) [inline test/runtests.jl]: Doesn't make much sense to use both a version conditional and Compat, one or the other would be simpler.

Comment (staticfloat) [reply test/runtests.jl]: The version of `Compat` that gets installed for 0.5 doesn't have the `Random` module; my guess is that `Compat` forked once it started handling the big parser changes for 0.7, and so 0.5 isn't getting these newer changes.

Comment (tkelman) [reply test/runtests.jl]: Then either drop 0.5 here, or don't go through Compat, use the specific Random stdlib version cutoff

**Comment by ararslan** [comment]
Superseded by #141.

**Pull request closed without merging**